contracts/svm/programs/express_relay/src/sdk/cpi.rs

```diff
@@ -0,0 +1,29 @@
+use {
+    crate::id as EXPRESS_RELAY_PID,
+    anchor_lang::prelude::*,
+    anchor_syn::codegen::program::common::sighash,
+};
+
+// Helper method to create a CPI to the Express Relay program to check permission for a given permission key and router
+pub fn check_permission<'info>(
+    sysvar_instructions: anchor_lang::solana_program::account_info::AccountInfo<'info>,
+    permission: anchor_lang::solana_program::account_info::AccountInfo<'info>,
+    router: anchor_lang::solana_program::account_info::AccountInfo<'info>,
+) -> Result<()> {
```

Comment:
any specific reason not return the instruction similar to what you did in the other file and also other crates?

I guess this way we can get rid of anchor all together in this file

Reply:
the use case here is not to be used in testing to construct a transaction/instruction. this is to be used to make the specified CPI into express relay and that alone, so it made sense to just make the call for them and return the output